### PR TITLE
[Doc]: Adjusting document presentation

### PR DESCRIPTION
**Pull request by mklzl**
remove some unused query and adjusting document presentation

## What type of PR is this?

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [x] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #4903 

## What this PR does / why we need it:

### CODE REVIEW

File: docs/cn/MatrixOne/Reference/SQL-Reference/Data-Manipulation-Statements/subqueries/comparisons-using-subqueries.md [hunk 1]
```diff
@@ -26,21 +26,20 @@ non_subquery_operand comparison_operator (subquery)
...
-|    7 |
 +------+
-2 rows in set (0.01 sec)
+1 row in set (0.00 sec)
 ```
 
 ## **限制**
 
-MatrixOne 暂不支持选择多列进行子查询。
+MatrixOne 在子查询与比较操作符结合使用时，暂不支持子查询结果为多行的操作。
```
Comment (lacrimosaprinz) [inline docs/cn/MatrixOne/Reference/SQL-Reference/Data-Manipulation-Statements/subqueries/comparisons-using-subqueries.md:45]: 当前修改仅适用示例中的情况；保留原文中的【限制】描述

File: docs/cn/MatrixOne/Reference/SQL-Reference/Data-Manipulation-Statements/subqueries/comparisons-using-subqueries.md [hunk 2]
```diff
@@ -26,19 +26,18 @@ non_subquery_operand comparison_operator (subquery)
 > create table t1 (a int);
 > create table t2 (a int, b int);
 > create table t3 (a int);
-> create table t4 (a int not null, b int not null);
 > insert into t1 values (2);
 > insert into t2 values (1,7),(2,7);
-> insert into t4 values (4,8),(3,8),(5,9);
 > insert into t3 values (6),(7),(3);
 > select * from t3 where a = (select b from t2);
+ERROR 1105 (HY000): scalar subquery returns more than 1 row
```
Comment (lacrimosaprinz) [inline docs/cn/MatrixOne/Reference/SQL-Reference/Data-Manipulation-Statements/subqueries/comparisons-using-subqueries.md:33]: @dengn  楠哥看看这个例子要不要等#3312的issue解决以后再增加？试了几个例子都存在一样的问题

Comment (dengn) [reply docs/cn/MatrixOne/Reference/SQL-Reference/Data-Manipulation-Statements/subqueries/comparisons-using-subqueries.md:33]: This is still a doc for 0.5.1, we need to add these examples telling others what are the limits and errors. We'll keep this one and add another one with multiple columns subquery with error message.

Comment (lacrimosaprinz) [reply docs/cn/MatrixOne/Reference/SQL-Reference/Data-Manipulation-Statements/subqueries/comparisons-using-subqueries.md:33]: comparisons subqueries does not support selecting multiple columns for the subquery. error message as below:
```
> create table t1 (a int);
> create table t2 (a int, b int);
> create table t3 (a int);
> create table t4 (a int not null, b int not null);
> insert into t1 values (2);
> insert into t2 values (1,7),(2,7);
> insert into t4 values (4,8),(3,8),(5,9);
> insert into t3 values (6),(7),(3);
> select a,b from t4 where a > ( select a ,b from t2 where a>1);
ERROR 1105 (HY000): Internal error: Unknow type TUPLE
```

Subqueries with ALL does not support selecting multiple columns for the subquery for 0.5.1. error message as below:
( I will add another example to the docs: "Subqueries with ALL/ANY/IN")
```
> create table t1 (a int);
> create table t2 (a int, b int);
> create table t3 (a int);
> create table t4 (a int not null, b int not null);
> create table t5 (a int);
> create table t6 (a int, b int);
> insert into t1 values (2);
> insert into t2 values (1,7),(2,7);
> insert into t4 values (4,8),(3,8),(5,9);
> insert into t5 values (null);
> insert into t3 values (6),(7),(3);
> insert into t6 values (10,7),(null,7);
> select a,b from t6 where a > all ( select a ,b from t4 where a>3);
ERROR 1105 (HY000): subquery should return 1 column
```

**Review by dengn — APPROVED** [review]

**Comment by dengn** [comment]
@mergifyio refresh

**Comment by dengn** [comment]
https://github.com/Mergifyio refresh

**Comment by yingfeng** [comment]
@Mergifyio refresh

**Pull request merged**